charts/consul/test/unit/client-daemonset.bats

```diff
@@ -543,7 +543,7 @@ load _helpers
 #--------------------------------------------------------------------
 # config-configmap
 
-@test "client/DaemonSet: adds config-checksum annotation when extraConfig is blank" {
```

Comment:
just re-worded the test titles a bit here because it *always* adds the config-checksum annotation so really what we're testing is that the checksum changes when the configmap is different